Add django, isa-l and libaom files (#138)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/libaom.md

---

> name: libaom

Author: ```suggestion
name: Libaom
```

category: Video
description: Libaom is an open-source codec library developed by the Alliance for Open Media (AOMedia).
download_url: https://github.com/ultrawide/libaom/tags
works_on_arm: true
supported_minimum_version:
    version_number: 1.0.0
    release_date: 25/06/2018

optional_info:
    homepage_url: https://aomedia.googlesource.com/aom/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content: 
        partner_content: 

>         official_docs: https://github.com/darkskygit/libaom/blob/v1.0.0/README.md

Author: ```suggestion
        official_docs: https://github.com/darkskygit/libaom/blob/master/README.md
```

    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: No arm64 specific release notes are available.Picked its 1.0.0 version as it is mentioned clearly in README.md file that this version has the support of AARCH64.

Author: ```suggestion
    other_info: Cross compilation support for arm64 is available from 1.0.0 version and steps are mentioned mentioned README of 1.0.0 tag release.
```
